docs/content/tutorial/saving-your-data.mdx

```diff
@@ -0,0 +1,179 @@
+
+You likely want to store your assets in places such as tables in your database or files in cloud storage. To do this, Dagster has I/O managers that enable reading and writing data to storage systems.
+
+By the end of this section, you will:
+
+- Store your files in a file system
+- Store your data frames as tables in a database
+- Learn about I/O managers
+
+---
+
+## What are I/O managers?
+
+I/O stands for **input** and **output.** I/O managers are Dagster objects that control how Dagster reads and writes data to specific external services, such as Snowflake or Amazon Web Services (AWS) S3.
+
+The **input** they manage is reading an asset from where it’s stored and loading it into memory to be used by a dependent asset. For example, `topstories_word_cloud` needs `topstories`. Therefore, the I/O manager that `topstories` uses will load the data into memory so the `topstories_word_cloud` asset can use it.
+
+The **output** that I/O managers control is writing to the location specified by the asset. By the end of this section, you’ll configure an I/O manager to write the `hackernews_topstories` DataFrame into a database as a table.
+
+---
+
+## Step 1: Writing files to storage
+
+Now, you’ll write your files to a more permanent location. For this tutorial, you’ll be writing to another directory in your file system by using the `fs_io_manager`, which is bundled with the core `dagster` package.
+
+In `__init__.py`, add the following snippet anywhere above the `def = Definitions(...)` line:
+
+```python
+from dagster import (
+    AssetSelection,
+    Definitions,
+    ScheduleDefinition,
+    define_asset_job,
+    load_assets_from_modules,
+    fs_io_manager # Update the imports at the top of the file to also include this
+)
+
+io_manager = fs_io_manager.configured({
+    "base_dir": "data", # Path is built relative to where `dagster dev` is run
+})
+```
+
+Next, update your `Definitions` (shown below) to attach the I/O manager to your code location by defining it as a **resource**.
+
+```python
+defs = Definitions(
+    assets=hackernews_assets,
+    schedules=[hackernews_schedule],
+    resources={
+        "io_manager": io_manager,
+    }
+)
+```
+
+<Note>
+  <strong>About resources</strong>
+  <br />
+  I/O managers are a special type of resource, a Dagster concept for{" "}
+  <strong>connecting</strong> to external services. I/O managers are resources
+  made specifically to control reading and writing to external services. Refer
+  to the <a href="/concepts/resources">Resources documentation</a> for more
+  information.
+</Note>
+
+To see the effect of this change, reload your code location in the Dagster UI and materialize your assets. When you click on an asset, you’ll see the `path` metadata pointing to a local storage directory.
```

Comment:
Even without this, it points to a local storage directory, right? We should explain what's different.